pom.xml

```diff
@@ -344,7 +345,13 @@
                 <artifactId>slf4j-log4j12</artifactId>
                 <version>${slf4j.version}</version>
             </dependency>
-            
+
+            <dependency>
+                <groupId>log4j</groupId>
+                <artifactId>log4j</artifactId>
+                <version>${log4j.version}</version>
+            </dependency>
+
```

Comment:
we don't need to import this, `slf4j-log4j12` already includes this.

Reply:
+1